lib/jellyfish_web/socket.ex

```diff
@@ -19,36 +19,32 @@ defmodule JellyfishWeb.Socket do
     with {:ok, peer_id} <- Map.fetch(state.params, "peer_id"),
          {:ok, room_id} <- Map.fetch(state.params, "room_id"),
          {:ok, room_pid} <- RoomService.find_room(room_id),
+         :ok <- Logger.metadata(room_id: room_id),
```

Comment:
We shouldn't do that. `room_id` should only be included in logs of room process and all its children